src/components/interaction-designer/block/BlockExit.vue

```diff
@@ -0,0 +1,341 @@
+            @lineMouseOut="setIsLineHovered(false)" />
+        </template>
+
+        <slot name="right" />
+      </span>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {computed, defineEmits, defineProps, nextTick, ref} from 'vue'
+import {IBlock, IBlockExit} from '@floip/flow-runner'
+import {
+  generateConnectionLayoutKeyFor,
+  OperationKind,
+  SupportedOperation,
+} from '@/store/builder'
+import {BLOCK_RESET_CONNECTIONS} from '@/components/interaction-designer/block/Block.vue'
+import {colorStates} from '@/components/interaction-designer/Connection.vue'
+import {useStore} from '@/store/useStore'
+import {lang} from '@/lib/filters/lang'
+
+const store = useStore()
+
+type Draggable = any
+const connectionColorAtSourceDragged = colorStates.CONNECTING
+const connectionColorForKnowDestination = colorStates.DEFAULT
+const ICON_SIZE = 10
+
+const props = defineProps<{
+  block: IBlock,
+  exit: IBlockExit,
+}>()
+
+// eslint-disable-next-line no-spaced-func,func-call-spacing
+const emit = defineEmits<{
+  (e: 'connection-create-start'): void,
+  (e: 'connection-create-end'): void,
+}>()
+
+const isDragged = ref(false)
+const isHovered = ref(false)
+const isLineHovered = ref(false)
+const isLinePermanentlyActive = ref(false)
+const livePosition = ref<{x: number, y: number} | null>(null)
+const cursorPosition = ref<{x: number, y: number} | null>(null)
+const componentKey = ref(0)
+
+const isEditable = computed<boolean>(() => store.getters['builder/isEditable'])
+
+const operations = computed<Record<OperationKind, SupportedOperation>>(() =>
+  store.state.builder.operations)
+
+const isConnectionCreateActive = computed(() =>
+  operations.value[OperationKind.CONNECTION_CREATE].data !== null)
+
+const isExitActivatedForCreate = computed(() => {
+  const {data} = operations.value[OperationKind.CONNECTION_CREATE]
+  return data?.source?.exitId === props.exit.uuid
+})
+
+const tooltip = computed<string>(() => {
+  if (!isEditable.value) {
+    return ''
+  } else if (props.exit.destination_block !== undefined) {
+    return lang.trans('flow-builder.tooltip-remove-connection')
+  } else {
+    return lang.trans('flow-builder.tooltip-new-connection')
+  }
+})
+
+const exitNameClass = computed(() => ({
+  'is-initiating': props.exit.destination_block == null && isDragged.value,
+  'is-connected': props.exit.destination_block != null,
+  'is-disconnected': props.exit.destination_block == null && !isDragged.value,
+  'is-highlighted-from-connection': props.exit.destination_block != null
+    && (isLineHovered.value || isLinePermanentlyActive.value),
+  'is-connected-and-on-hover': props.exit.destination_block != null && isHovered.value,
+}))
+
+const blocksById = computed<Record<IBlock['uuid'], IBlock>>(() =>
+  store.getters['builder/blocksById'])
+
+const draggableForExitsByUuid = computed<Record<string, Draggable>>(() =>
+  store.state.builder.draggableForExitsByUuid)
+
+function forceRender() {
```

Comment:
I didn't know about this trick. Thanks for fixing 👍

Reply:
You're welcome, I found it from a forum as well :D